src/renderer/views/Playlist/Playlist.js

```diff
@@ -280,6 +284,7 @@ export default defineComponent({
         } else {
           this.showUserPlaylistNotFound()
         }
+        this.checkPlaylist()
```

Comment:
Should this be called if user playlist not found, I guess not

Reply:
Yeah, probably not, I'll move up to only if the playlist is found.